x-pack/winlogbeat/module/sysmon/config/winlogbeat-sysmon.js

```diff
@@ -17,74 +17,45 @@ var sysmon = (function () {
     var processor = require("processor");
     var winlogbeat = require("winlogbeat");
 
-    // Windows error codes for DNS.
-    // https://docs.microsoft.com/en-us/windows/win32/debug/system-error-codes--9000-11999-
+    // Windows error codes for DNS. This list was generated using
+    // 'go run gen_dns_error_codes.go'.
     var dnsQueryStatusCodes = {
         "0": "SUCCESS",
-        "2329": "DNS_ERROR_RCODE_FORMAT_ERROR",
```

Comment:
I'm not sure where I got these codes from originally because they don't seem to exist 🤷‍♂ .

Reply:
This is really weird. The ID for all this removed error codes is in hex, while they have been re-added in decimal. For example:
```diff
-       "2330": "DNS_ERROR_RCODE_NXRRSET",
+       "9008": "DNS_ERROR_RCODE_NXRRSET",
```
> DNS_ERROR_RCODE_NXRRSET 9008 (0x2330)

And all the errors where the hex code included an non-decimal hex digit were missing.